src/price_data.py

```diff
@@ -97,7 +97,7 @@ def _get_price_binance(
             and data.get("code") == -1121
             and data.get("msg") == "Invalid symbol."
         ):
-            if quote_asset == "BTC":
+            if quote_asset == "BTC" or base_asset == "BTC":
```

Comment:
When does the case base_asset == "BTC" happen?

Reply:
I should've investigated this right away as I cannot reproduce it anymore. 

I can imagine it was some kind of issue I had in combination with some local changes where I used Binance as a fallback for Coinbase pairs (e.g. because XRP is not part of the Coinbase API anymore).

I've reverted the change.